ui/packages/app/src/components/File/File.svelte

```diff
@@ -52,6 +52,6 @@
 			upload_text={$_("interface.click_to_upload")}
 		/>
 	{:else}
-		<File value={_value} {label} {show_label} {file_count} />
```

Comment:
Why did we remove `file_count`? I thought it was needed to be able to upload multiple files or a directory

Reply:
Only actually used in the `FileUpload` component, to determine what can be uploaded.